helpdesk/overview/getting_started.rst

```diff
@@ -1,171 +1,67 @@
 
-The final option to submit tickets is through an API connection. View the
-documentation `here <https://www.odoo.com/documentation/13.0/webservices/odoo.html>`_.
-
-Tickets have been created, now what?
-------------------------------------
-
-Now your employees can start working on them! If you have selecting a
-manual assignation method then your employees will need to assign
-themselves to tickets using the "Assign To Me" button on the top left of
-a ticket or by adding themselves to the "Assigned to" field.
-
-.. image:: media/helpdesk09.png
-   :align: center
-
-If you have selected "Random" or "Balanced" assignation method, your
-tickets will be assigned to a member of that Helpdesk team.
-
-From there they will begin working on resolving the tickets! When they
-are completed, they will move the ticket to the solved stage.
-
-How do I mark this ticket as urgent?
-------------------------------------
-
-On your tickets you will see stars. You can determine how urgent a
-ticket is but selecting one or more stars on the ticket. You can do this
-in the Kanban view or on the ticket form.
-
-.. image:: media/helpdesk10.png
-   :align: center
-
-To set up a Service Level Agreement Policy for your employees, first
-activate the setting under "Settings"
-
-.. image:: media/helpdesk11.png
-   :align: center
+Team’s productivity and visibility
+----------------------------------
 
-From here, select "Configure SLA Policies" and click "Create".
+| Teams can have individual *Assignment Methods* to ensure tickets get redirected to the right
+  person:
 
-You will fill in information like the Helpdesk team, what the minimum
-priority is on the ticket (the stars) and the targets for the ticket.
+- *Manually*: tickets are manually assigned allowing employees to manage their own workload and
+  target tickets they are experts at;
+- *Random*: tickets are randomly assigned and everyone gets the same amount. This method ensures
+  that all tickets are handled as the assignment happens automatically;
+- *Balanced*: tickets are assigned to the person with the least amount of tickets so that everyone
+  fairly gets the same amount. Thereby, you ensure that all tickets get to be taken care of, as
+  the assignment is automatic.
 
-.. image:: media/helpdesk12.png
+.. image:: media/productivity_visibility.png
    :align: center
+   :alt: View of a helpdesk team settings page emphasizing the productivity and visibility features
+         in Odoo Helpdesk
 
-What if a ticket is blocked or is ready to be worked on?
---------------------------------------------------------
-
-If a ticket cannot be resolved or is blocked, you can adjust the "Kanban
-State" on the ticket. You have 3 options:
-
-- Grey - Normal State
+| For the *Random* and *Balanced* assignment methods, you are able to set the *Team Members* among
+  whom tickets are assigned. Leave the field empty to include all employees (with the proper
+  access rights).
```

Comment:
```suggestion
| For the *Random* and *Balanced* assignment methods, you can set the *Team Members* among
  whom tickets are assigned. Leave the field empty to include all employees (with the proper
  access rights).
```

Reply:
Later: it would be good to add a link to the access rights doc, but maybe you already planned it ^^

Reply:
yes